backend/compact-connect/stacks/api_stack/v1_api/staff_users.py

```diff
@@ -0,0 +1,417 @@
+            },
+            authorization_type=AuthorizationType.COGNITO,
+            authorizer=self.api.staff_users_authorizer,
+            authorization_scopes=admin_scopes
+        )
+
+    def _add_get_user(self, user_id_resource: Resource, admin_scopes: List[str]):
+        with open(os.path.join('lambdas', 'staff-users', 'tests', 'resources', 'me.json')) as f:
+            response_template = f.read()
+
+        user_id_resource.add_method(
+            'GET',
+            request_validator=self.api.parameter_body_validator,
+            method_responses=[
+                MethodResponse(
+                    status_code='200',
+                    response_models={
+                        'application/json': self.get_me_model
+                    }
+                )
+            ],
+            integration=MockIntegration(
+                integration_responses=[
+                    IntegrationResponse(
+                        status_code='200',
+                        response_templates={
+                            'application/json': response_template
+                        }
+                    )
+                ]
+            ),
+            request_parameters={
+                'method.request.header.Authorization': True
+            },
+            authorization_type=AuthorizationType.COGNITO,
+            authorizer=self.api.staff_users_authorizer,
+            authorization_scopes=admin_scopes
+        )
+
+    def _add_patch_user(self, me_resource: Resource, admin_scopes: List[str]):
+        with open(os.path.join('lambdas', 'staff-users', 'tests', 'resources', 'me.json')) as f:
+            response_template = f.read()
+
+        me_resource.add_method(
+            'PATCH',
+            request_validator=self.api.parameter_body_validator,
+            request_models={
+                'application/json': self.patch_user_model
+            },
+            method_responses=[
+                MethodResponse(
+                    status_code='200',
+                    response_models={
+                        'application/json': self.get_me_model
+                    }
+                )
+            ],
+            integration=MockIntegration(
+                integration_responses=[
+                    IntegrationResponse(
+                        status_code='200',
+                        response_templates={
+                            'application/json': response_template
+                        }
+                    )
+                ]
+            ),
+            request_parameters={
+                'method.request.header.Authorization': True
+            },
+            authorization_type=AuthorizationType.COGNITO,
+            authorizer=self.api.staff_users_authorizer,
+            authorization_scopes=admin_scopes
+        )
+
+    @cached_property
+    def get_me_model(self):
+        """
+        Return the Get Me Model, which should only be created once per API
```

Comment:
Would it make sense to separate these JSON Schema model definitions into a separate schema folder/file, to help reduce the file size here?

Reply:
Yeah, possibly. TBH, I haven't entirely decided how I want to organize the API schema yet.